terraform/environments/electronic-monitoring-data/bastion_linux.tf

```diff
@@ -36,3 +36,56 @@ module "rds_bastion" {
   tags_common = local.tags
   tags_prefix = terraform.workspace
 }
+
+resource "aws_vpc_security_group_egress_rule" "access_ms_sql_server" {
+  security_group_id = module.rds_bastion.bastion_security_group
+  description       = "EC2 MSSQL Access"
+  ip_protocol       = "tcp"
+  from_port         = 1433
+  to_port           = 1433
+  cidr_ipv4         = data.aws_vpc.shared.cidr_block
+}
+
+resource "aws_vpc_security_group_ingress_rule" "rds_via_vpc_access" {
+  security_group_id = aws_security_group.db.id
+  description       = "EC2 instance connection to RDS"
+  ip_protocol       = "tcp"
+  from_port         = 1433
+  to_port           = 1433
+  referenced_security_group_id = module.rds_bastion.bastion_security_group
+}
+
```

Comment:
do we not need this?
```tf
resource "aws_vpc_security_group_egress_rule" "vpc_access" {
  security_group_id = module.rds_bastion.bastion_security_group.id
  description       = "Reach vpc endpoints"
  ip_protocol       = "tcp"
  from_port         = 443
  to_port           = 443
  cidr_ipv4         = data.aws_vpc.shared.cidr_block
}
```

Reply:
I don't think so? Connecting to rds with azure data studio and querying is working without. When things fail in prod will revisit